.env.example

```diff
@@ -20,3 +20,8 @@ VITE_USE_FIAT_EXCHANGE_RATES=true
 VITE_FIAT_EXCHANGE_RATES_API_URL=https://api.exchangeratesapi.io/v1/latest
 VITE_FIAT_EXCHANGE_RATES_API_KEY=
 VITE_FIAT_EXCHANGE_RATES_ETHEREUM_USDC_ADDRESS=0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
+
+# MISC
```

Comment:
I'd add a comment here:

`# Both the title and the message are required for the message to be displayed.`

Reply:
In fact I also consider making the title optional and that only the message is required to display the popup.

Reply:
Then the message could be:

`# VITE_OUTDATED_NETWORK_MESSAGE is required for the popup to be displayed.`

Reply:
nah I don't think the comment is required now